client/src/main/scala/caliban/client/Selection.scala

```diff
@@ -17,11 +17,12 @@ object Selection {
   case class Directive(name: String, arguments: List[Argument[_]] = Nil) {
     def toGraphQL(
       useVariables: Boolean,
+      dropInputNullValues: Boolean,
```

Comment:
Nit: I think it would read better as `dropNullInputValues`

Reply:
Done~